mlir/lib/Dialect/XeGPU/IR/XeGPUDialect.cpp

```diff
@@ -96,6 +107,16 @@ void TensorDescType::print(::mlir::AsmPrinter &printer) const {
   printer << ">";
 }
 
+TensorDescType TensorDescType::get(mlir::MLIRContext *context,
+                                   llvm::ArrayRef<int64_t> shape,
+                                   mlir::Type elementType, bool scattered,
```

Comment:
Since you're taking a Type as argument here, you don't need to also take a context.

Instead of a `TypeBuilder` you just have to declare it as `TypeBuilderWithInferredContext`

Reply:
Fixed it.